### ISSUE TITLE
[prometheus-metrics-adapter] Update HPA docs to autoscaling/v2

### ISSUE THREAD

**Issue by apolovov** [deckhouse/deckhouse#2759]
### Preflight Checklist

- [X] I agree to follow the [Code of Conduct](https://github.com/deckhouse/deckhouse/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/deckhouse/deckhouse/issues) for an issue that matches the one I want to file, without success.

### Use case. Why is this important?

The docs suggest us to use `apiVersion: autoscaling/v2beta2`, but the actual one is autoscaling/v2.

### Proposed Solution

To figure out if there are some changes, update docs and test them.

**Comment by duckhawk** [comment]
v2 has been available since 1.23 and default deckhouse version now is 1.21. I'm afraid this can lead confusion among users. Maybe we'll wait until default version will be 1.23?

**Comment by gyrter** [comment]
Up! Deckhouse queue hungs if you update to 1.26 on ingress-nginx module.

```
 1. ModuleRun:main:ingress-nginx:doStartup:OperatorStartup:failures 5:unable to build kubernetes objects from release manifest: resource mapping not found for name: "hpa-scaler-main" namespace: "d8-ingress-nginx" from "": no matches for kind "HorizontalPodAutoscaler" in version "autoscaling/v2beta2"
ensure CRDs are installed first
```
